src/main/java/mate/jdbc/service/CarServiceImpl.java

```diff
@@ -48,12 +48,11 @@ public void addDriverToCar(Driver driver, Car car) {
     @Override
     public void removeDriverFromCar(Driver driver, Car car) {
         if (car.getDrivers() != null) {
```

Comment:
missed the change in the condition, it should be == null, now that we throw an exception in its body